zrml/prediction-markets/src/lib.rs

```diff
@@ -214,19 +215,20 @@ mod pallet {
         ) -> DispatchResultWithPostInfo {
             T::ApprovalOrigin::ensure_origin(origin)?;
 
-            let market = Self::market_by_id(&market_id)?;
+            let market = T::MarketCommons::market(&market_id)?;
             ensure!(
                 market.status == MarketStatus::Reported || market.status == MarketStatus::Disputed,
                 "not reported nor disputed"
             );
             Self::clear_auto_resolve(&market_id)?;
-            let weight = Self::internal_resolve(&market_id)?;
-            Ok(Some(
-                weight
-                    .saturating_add(T::WeightInfo::admin_move_market_to_resolved_overhead())
-                    .saturating_sub(T::WeightInfo::internal_resolve_scalar_reported()),
-            )
-            .into())
+            Ok(Some(0u64).into())
+            //let weight = Self::internal_resolve(&market_id)?;
+            //Ok(Some(
+            //    weight
+            //        .saturating_add(T::WeightInfo::admin_move_market_to_resolved_overhead())
+            //        .saturating_sub(T::WeightInfo::internal_resolve_scalar_reported()),
+            //)
+            //.into())
```

Comment:
These and other benchmarks comments will be evaluated once https://github.com/zeitgeistpm/zeitgeist/pull/167#discussion_r652274415 is resolved